visualization/R/add_icon.R

```diff
@@ -1,18 +1,19 @@
 ##' @name add_icon
 ##' @param x x-coordinate of logo
 ##' @param y y-coordinate of logo
+##' @param id additional plot identificaition (URL, database ID, etc)
 ##' @export 
 ##' @author Mike Dietze
 ##' 
 add_icon <- function(id=NULL,x=0,y=0){
-  require(png)
-  require(grid)
+  include(png)
+  include(grid)
```

Comment:
Sorry to be a pain, but what's `include`? (Code is not Python.) Shouldn't these two lines use `library`?